client/state/sites/reducer.js

```diff
@@ -214,6 +226,32 @@ export function items( state = {}, action ) {
 
 			return state;
 		}
+
+		case InvitesActionTypes.RECEIVE_INVITE_ACCEPTED_SUCCESS: {
+			const { invite, data } = action;
+
+			if ( ! invite || ! data || ! data.sites ) {
+				return state;
+			}
+
+			if ( invite.role === 'follower' || invite.role === 'viewer' ) {
+				return state;
+			}
+
+			const siteId = get( invite, 'site.ID', null );
+
+			if ( ! siteId || state[ siteId ] ) {
```

Comment:
`siteId` is the ID of the site to which you just accepted an invitation. Here you check if that site is already present in `state`. But it's not there yet -- it will be added by this action.

Reply:
I check it just in case the user was already invited to the site or for any rare case the site already exists in the state.